spec/engine/miq_ae_method_service/miq_ae_service_methods_spec.rb

```diff
@@ -437,17 +437,22 @@ def category_create_script_kwargs
              :persist_state_hash => MiqAeEngine::StateVarHash.new,
              :ae_user            => user)
     end
-    let(:user) { double }
-    let(:miq_ae_service) { MiqAeMethodService::MiqAeService.new(workspace) }
+    let(:user)            { double }
+    let(:miq_ae_service)  { MiqAeMethodService::MiqAeService.new(workspace) }
+    let(:version)         { "1.1" }
+    let(:template_fields) { {"name" => "App"} }
+    let(:vm_fields)       { {'vm_name' => 'CRM_APP', 'request_type' => 'template'} }
+    let(:requester)       { {'owner_email' => 'admin@asd.com'} }
+    let(:tags)            { {'crm' => 'true'} }
     before do
       allow(User).to receive(:lookup_by_userid).and_return(user)
     end
 
     it "passes arguments correctly" do
-      expect(MiqProvisionVirtWorkflow).to receive(:from_ws).with('one', 'two', user).and_return(true)
+      expect(MiqProvisionVirtWorkflow).to receive(:from_ws).with(version, user, template_fields, vm_fields, requester, "ws_values", "ems_custom_attributes", "miq_custom_attributes").and_return(true)
       expect(MiqAeMethodService::MiqAeServiceModelBase).to receive(:wrap_results).and_return(true)
       allow(workspace).to receive(:disable_rbac)
-      miq_ae_service.execute(:create_provision_request, 'one', 'two')
+      miq_ae_service.execute(:create_provision_request, version, template_fields, vm_fields, requester, "ws_values", "ems_custom_attributes", "miq_custom_attributes")
     end
 
     it "passes array arguments correctly" do
```

Comment:
So none of the following sets of arguments are valid, but rather than delete them I figured that this isn't testing `MiqProvisionVirtWorkflow.from_ws` it is just testing `MiqAeServiceMethods.create_provision_task`.  This tests how the arguments will be passed through to the underlying method.